mucgpt-frontend/src/components/LanguageSelector/LanguageSelector.module.css

```
@@ -1,43 +1,172 @@
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.fallbackCode {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)